### ISSUE TITLE
Jester eval script

### ISSUE THREAD

**Issue by hugormluis** [okankop/Efficient-3DCNNs#20]
Hello, I want to evaluate the video accuracy on the jester dataset although there isn't any script for that. I only found for UCF and kinetics.

Another question, during the train process what's the meaning of the top 1 accuracy vs the top 1 accuracy during the validation? Why are they different? For example, for the jester dataset typically the validation accuracy is higher (15% to 20%) than the train top 1 accuracy.

**Comment by okankop** [comment]
There is no script needed for video acc for jester. Because, avg video duration is around 34-36 frames for jester, and when you use 32 frames input (or 16 frames input with downsampling of 2), the clip already covers the entire video.

There is a difference at training and validation accuracy, because an intensive data augmentation is applied during training.

**Comment by hugormluis** [comment]
Ok, so what was that accuracy that you used for your paper? The validation accuracy? I trained the mobilenetV1 0.5x from scratch and used this steps for the lr: [25, 40, 50, 65] and stopped at the 80th epoch. I got top1 train accuracy of 72% and validation accuracy of 86%. In this case it went accordingly to the expectations and the 86% accuracy is the one that you presented in the table you showed in the paper?

Thank you for your fast and comprehensive response!
